types/sharedb/index.d.ts

```diff
@@ -326,4 +332,17 @@ interface GetOpsOptions {
     };
 }
 
+interface PresenceMessage {
+    a: 'p';
+    ch: string; // channel
+    src: string; // client ID
+    id: string; // presence ID
+    p: any; // presence payload
+    pv: number; // presence version
+    c?: string; // document collection
+    d?: string; // document ID
+    v?: number; // document number
+    t?: string; // document type
```

Comment:
I'd suggest going with `// document version` and `// document OT type` to make their meanings clearer. Though we don't even have full comments on these in our JS source 😅

Reply:
I actually meant to write `version`, just had a brain burp. Good catch.